src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,166 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * This class deals with loading tasks from saved file and
+ * saving tasks in the file after the job is done.
+ */
+public class Storage {
+
+    public File store;
```

Comment:
Should this variable name be something like "file" instead as it better describes the variable?